File: front/chatapp/src/app/model/ChatMessage.ts
export class ChatMessage {

    constructor(
        public id: string,
        public chatId: string,
        public senderId: string,
        public recipientId: string,
        public senderName: string,
        public recipientName: string,
        public content: string,
        public timestamp: Date,
        public status: string) {

    }

}

File: back/chat-back/src/main/java/com/app/chatBack/model/entity/ChatMessage.java
package com.app.chatBack.model.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Document(collection = "chat_message")
public class ChatMessage {

   @Id
   private String id;
   private String chatId;
   private String clientId;
   private String consultantId;
   private String content;
   private Date timestamp;
   private String status;
   private String typeMessage;

}
